metricbeat/helper/prometheus/textparse.go

```diff
@@ -537,15 +546,19 @@ func ParseMetricFamilies(b []byte, contentType string, ts time.Time) ([]*MetricF
 		var lbls strings.Builder
 		lbls.Grow(len(mets))
 		var labelPairs = []*labels.Label{}
+		var qv string // value of le or quantile label
 		for _, l := range lset.Copy() {
 			if l.Name == labels.MetricName {
 				continue
 			}
 
-			if l.Name != model.QuantileLabel && l.Name != labels.BucketLabel { // quantile and le are special labels handled below
-
+			if l.Name != model.QuantileLabel && l.Name != labels.BucketLabel {
```

Comment:
it would be more readable if changed order or used `switch`

Reply:
I changed the order. Is it better now?